### ISSUE TITLE
About ImageNet

### ISSUE THREAD

**Issue by gugugei** [sunshangquan/logit-standardardization-KD#32]
I have read your paper and find it a very valuable contribution. I have a few questions I would like to ask:

How many GPUs did you use for training on ImageNet? Do you think the training results could vary depending on the number of GPUs used?

When training our own models, some of our results are not as good as expected, possibly due to suboptimal parameter settings. If the total batch size is set to 512 on 8 GPUs, would each GPU be allocated a batch size of 64?

Regarding the hyperparameters of the loss function, would you recommend setting the hyperparameters of our custom loss smaller than those used for the CE loss? I noticed that many papers keep them the same as the CE loss.

Thank you very much for your time and for answering my questions!

Best regards

**Comment by sunshangquan** [comment]
Hi! @gugugei Sorry that I did not try any setting of multiple GPUs, so I could not answer your second question based on my experience. For the custom loss, you could certainly try, though I do not know what loss you are using.

**Comment by gugugei** [comment]
Thanks for your reply. Did you use a GPU for training in mlkd?
